### PR TITLE
Add LEGO Education SPIKE article

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Add LEGO Education SPIKE article
- Associated [Google Doc](https://docs.google.com/document/d/18YclnkbFxPS1vOoT61IO5fBSSO7S6wxHmMbfIJyn6kw/edit?resourcekey=0-ed_79784iy6mxisK4rh6kQ#)
- [Staging preview](https://pr-6351-static-dot-dcc-staging.uc.r.appspot.com/blog/lego-education-spike-web-bluetooth-web-serial/)

### CODE REVIEW

File: site/en/blog/lego-education-spike-web-bluetooth-web-serial/index.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+tags:
+  - capabilities
+  - case-study
+  - fugu-case-study
+---
+
+The
+[LEGO® Education SPIKE™ Prime Set](https://education.lego.com/en-us/products/lego-education-spike-prime-set/45678#spike%E2%84%A2-prime)
+is a STEAM (science, technology, engineering, arts and mathematics) learning
+tool for grade 6–8 students. Combining colorful LEGO building elements,
```
[the remark this answers is not in the document]
Comment (rachelandrew) [reply site/en/blog/lego-education-spike-web-bluetooth-web-serial/index.md:23]: Sorry missed this, also might be worth adding the age? I'm not sure I know (as a Brit) what age that maps to so maybe in brackets (around ages n to y).

**Review by rachelandrew — APPROVED** [review]
LGTM with one note.

**Comment by tomayac** [comment]
It translates to about 11 to 13 years according to multiple sources.

File: site/en/blog/lego-education-spike-web-bluetooth-web-serial/index.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
+---
+layout: 'layouts/blog-post.njk'
+title: 'How LEGO® Education uses the Web Bluetooth and the Web Serial APIs'
+subhead: >
+  For its programmable SPIKE™ kits, LEGO® Education uses the power of the web to
+  make connecting the LEGO models to the computer as simple as possible.
+date: 2023-05-17
+# updated: 2023-05-17
```
Comment (tomayac) [inline site/en/blog/lego-education-spike-web-bluetooth-web-serial/index.md:8]:
```suggestion
date: 2023-05-22
# updated: 2023-05-22
```

**Pull request merged**